src/server/models/mediaquery.js

```diff
@@ -126,35 +126,44 @@ module.exports.getSubjectAreasForTerm = async (dbCollection, term) => {
   return subjectAreasArray;
 };
 
-module.exports.getVideoResByCourse = async courseLabel => {
-  const videoResCollection = client.db(DB_DATABASE).collection('videoreserves');
+module.exports.getVideoResByCourse = async (courseLabel, collectionName) => {
+  const videoResCollection = client.db(DB_DATABASE).collection(collectionName);
   const toBeReturned = await videoResCollection
     .find({ classShortname: courseLabel })
     .sort({ videoTitle: 1 })
     .toArray();
   return toBeReturned;
 };
 
-module.exports.getVideoResCountByCourse = async courseLabel => {
+module.exports.getVideoResCountByCourse = async (
+  courseLabel,
+  collectionName
+) => {
   const resCount = await client
     .db(DB_DATABASE)
-    .collection('videoreserves')
+    .collection(collectionName)
     .find({ classShortname: courseLabel })
     .count();
   return resCount;
 };
 
-module.exports.getMusicResByCourse = async courseLabel => {
-  const musicResCollection = client.db(DB_DATABASE).collection('musicreserves');
+module.exports.getMusicResByCourse = async (courseLabel, collectionName) => {
+  const musicResCollection = client.db(DB_DATABASE).collection(collectionName);
   const toBeReturned = await musicResCollection
     .find({ classShortname: courseLabel })
     .sort({ title: 1 })
     .toArray();
   return toBeReturned;
 };
 
-module.exports.getMusicResCountByCourse = async courseLabel => {
-  const arrayOfMusicRes = await this.getMusicResByCourse(courseLabel);
+module.exports.getMusicResCountByCourse = async (
+  courseLabel,
+  collectionName
```

Comment:
@maxpluez  is there a way to tell if you are in a jest test? If so then you can automatically add test to the collection name and avoid having to keep passing in collection name every time

Reply:
I don't think so. Private variables cannot be mocked.
https://stackoverflow.com/questions/48616566/jest-testing-variables-inside-component-methods
Here's a similar question, someone provided some solutions but according to the asker none of the solutions worked.
https://stackoverflow.com/questions/48006588/jest-mock-functions-inner-variable-and-change-functions-behavior

Reply:
@maxpluez No.. you don't mock it.. You have actual logic in the code that adds "test" to the collection name if it detects it is running in a jest test.

Maybe it can be as simple as setting process.env.MODE='test' in the beforeAll() and in the functions check if the MODE is test, then add "test' to the collection names.